wts/resources/oauth2.py

```diff
@@ -108,11 +109,18 @@ def refresh_refresh_token(tokens, idp, username_field):
             idp_username, idp, username
         )
     )
+    jti_value = content.get("jti")
+    if jti_value is None:
+        jti_value = str(uuid.uuid4())
+        flask.current_app.logger.warning(
+            f"Missing 'jti' field in token response for user {userid}. Generated UUID: {jti_value}"
```

Comment:
```suggestion
            f"Missing 'jti' field in token response for user {userid}. Generated UUID: {jti_value} and used as 'jti' in our db. Note that the token issuer is not directly made aware of this association."
```

Reply:
maybe that last sentence can be a comment. But just want future folks digging into this to realize that we're not "officially" assigning that JTI in a way that the issuer is aware. e.g. we can't go to them and say "hey we saw this UUID token in our logs from someone causing issues".